bookkeeper-server/src/main/java/org/apache/bookkeeper/server/Main.java

```diff
@@ -76,6 +76,7 @@
         BK_OPTS.addOption("z", "zkserver", true, "Zookeeper Server");
         BK_OPTS.addOption("m", "zkledgerpath", true, "Zookeeper ledgers root path");
         BK_OPTS.addOption("p", "bookieport", true, "bookie port exported");
+        BK_OPTS.addOption("httpport", true, "bookie http port exported");
```

Comment:
"hp" as short-hand ?

Reply:
makes sense

Reply:
Thanks for the suggestion, just added it.